OrasDotNet/Models/Descriptor.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+
+namespace OrasDotnet.Models
+{
+    public class Descriptor
+    {
+        [JsonPropertyName("mediaType")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public string MediaType { get; set; }
+
+        [JsonPropertyName("digest")]
+        public string Digest { get; set; }
+
+        [JsonPropertyName("size")]
+        public long Size { get; set; }
+
+        [JsonPropertyName("urls")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public List<string> URLs { get; set; }
+
+        [JsonPropertyName("annotations")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public Dictionary<string, string> Annotations { get; set; }
```

Comment:
How about `IDictionary`
```suggestion
        public IDictionary<string, string> Annotations { get; set; }
```